src/Microsoft.DocAsCode.App/DocfxProject.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DocAsCode.Common;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DocAsCode
+{
+    /// <summary>
+    /// Provides options to be used with <see cref="DocfxProject.Build(BuildOptions)"/>.
+    /// </summary>
+    public class BuildOptions
```

Comment:
Do we need a more specific name, e.g. `DocfxBuildOptions`?

Reply:
I would say no, but that is a good question on whether we should rename `DocfxProject` -> `Project`. The term project might be too general so another candidate name is `Docset`, which is widely used in mslearn.